recipe/meta.yaml

```diff
@@ -9,6 +9,8 @@ source:
   fn: python-graphviz-{{ version }}.zip
   url: https://pypi.io/packages/source/g/graphviz/graphviz-{{ version }}.zip
   sha256: {{ sha256 }}
+  patches:
+    - windows-bat.patch
 
 build:
   number: 0
```

Comment:
Can you bump the build number to 1 please?

Reply:
@scopatz Sure, done!